src/angular/ngVueLinker.js

```diff
@@ -48,7 +48,11 @@ export function ngVueLinker (componentName, jqElement, elAttributes, scope, $inj
 
   const vueInstance = mountReplace(Object.assign({}, vueHooks, config, {
     render () {
-      return h(Component, reactiveData);
+      return h(Component, reactiveData, {
```

Comment:
What if you set `reactiveData.innerHTML = html`? I tried it out in a Vue 3 playground and it seemed to work. i.e.
```
// component A
export const A = {
  render() { return h('span'); }
}

// component B
export const B = {
  render() { return h(A, { innerHTML: '<em>My raw HTML.</em>' }); }
};
```
using component `B` renders the raw html inside a span from component A.

Reply:
Tried it last night and no dice. The compatibility build and the peculiarities of our angular/vue hybrid make most documentation close to useless.

Reply:
:+1: